### DISCUSSION TITLE
Welcome to rio-tiler Discussions!

### DISCUSSION THREAD

**Discussion by vincentsarago** [cogeotiff/rio-tiler#318] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jqtrde** [comment]
Hi, I'm Jacques! I'm here because rio-tiler is nice, and other reasons.

Just one question @vincentsarago, who is your favorite Mainer?
